packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -322,7 +327,23 @@ onClickOutside(createBtn as Ref<VueInstance>, () => {
 				:menu="menu"
 				@select="handleMenuSelect"
 			>
-				<N8nIconButton icon="plus" type="secondary" outline />
+				<n8n-icon-button icon="plus" type="secondary" outline />
```

Comment:
Just a stylistic nitpick: we should use `PascalCase` for the design system components (you shouldn't have changed it and if you see otherwise somewhere else you can fix those as well)